netsim/ansible/tasks/deploy-config/cumulus_nvue.yml

```diff
@@ -13,8 +13,8 @@
     service_facts:
     register: svc
     until: svc.ansible_facts.services['nvued.service'].state == 'running'
-    retries: 10
-    delay: 5
+    retries: 5
```

Comment:
I prefer having smaller delay and more retries. On average, that reduces the waiting time.

Reply:
you got it ;)